code/game/machinery/recharger.dm

```diff
@@ -83,13 +74,6 @@ obj/machinery/recharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
 		G.loc = src
 		charging = G
 		update_icon()
-	else if(portable && istype(G, /obj/item/weapon/wrench))
```

Comment:
it was duplicate code, the same wrench interaction was stuck twice in the attackby proc, i just deleted the latter one. wrenching and unwrenching it still works fine